planetarypraxis/smartforests
# Add option to sort logbook entries

## Description
This PR adds an option to sort logbooks and logbook entries by date published and alphabetically by title

## Motivation and Context
Linked to issue [SF3-21](https://linear.app/commonknowledge/issue/SF3-21/create-an-option-to-sort-the-logbook-entries)

## How Can It Be Tested?
Download the branch and run locally. Navigate to logbook and logbook index pages. Make sure the CMS contains some logbooks and logbook entries. View the dropdown.

## Screenshots  (if appropriate):

<img width="1362" alt="Screenshot 2023-03-13 at 18 47 23" src="https://user-images.githubusercontent.com/43313455/224801066-8c6ef3ea-94df-4fb0-8b90-0dbb29194304.png">
<img width="593" alt="Screenshot 2023-03-13 at 18 47 40" src="https://user-images.githubusercontent.com/43313455/224801128-645f18e2-df6c-4cb8-a2c9-fb4b68681820.png">
<img width="1372" alt="Screenshot 2023-03-13 at 18 53 23" src="https://user-images.githubusercontent.com/43313455/224801537-77e313e6-435d-4e82-a702-65406847ccaa.png">

<img width="589" alt="Screenshot 2023-03-13 at 18 53 44" src="https://user-images.githubusercontent.com/43313455/224801521-216d1259-9814-4de4-bbbb-e9102eb77c01.png">


## Types of changes
